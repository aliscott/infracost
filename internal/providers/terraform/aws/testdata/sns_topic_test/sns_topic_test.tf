provider "aws" {
  region                      = "us-east-1"
  skip_credentials_validation = true
  skip_requesting_account_id  = true
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
}

resource "aws_sns_topic" "sns_topic" {
  name = "my-standard-queue"
}

resource "aws_sns_topic" "sns_topic_withUsage" {
  name = "my-standard-queue"
}