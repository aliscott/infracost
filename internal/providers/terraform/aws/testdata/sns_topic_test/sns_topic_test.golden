
 Name                                  Monthly Qty  Unit              Monthly Cost 
                                                                                   
 aws_sns_topic.sns_topic                                                           
 └─ Requests                        Cost depends on usage: $0.50 per 1M requests   
                                                                                   
 aws_sns_topic.sns_topic_withUsage                                                 
 └─ Requests                                     2  1M requests              $1.00 
                                                                                   
 PROJECT TOTAL                                                               $1.00 

----------------------------------
To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file